hazelcast/src/test/java/com/hazelcast/dataconnection/impl/ListResourcesTest.java

```diff
@@ -97,6 +96,8 @@ public void shouldListResourcesDefaultAndNonDefaultSchema() throws Exception {
         );
 
         executeJdbc(jdbcUrl, "CREATE TABLE my_table (id INTEGER, name VARCHAR(255) )");
+        // In MySQL, the CREATE SCHEMA statement is synonymous with the CREATE DATABASE statement.
+        // We should not be able to list resources in another database
```

Comment:
But we should be able to list resources from another schema.  

You can also say that database behaves like a schema in MySQL and there is nothing like a real database.

So removing the `                                new DataConnectionResource("TABLE", "my_schema", "my_table")` expected resource is wrong.

Reply:
We are assuming that a schema is same as a database. We should not list resources for a different db. We don't do it for for example for Postgres. If the user wants to list a separate DB, he needs to create a separate connection.

Reply:
Ok for mysql it is listing everything now. Also added SQL Server test 
rev : 0e7e257b53a2d664d08ef709119dbe17ade54e01